src/config/env/prod.json

```diff
@@ -1,3 +1,6 @@
 {
-  "CHAIN_ID": "1"
-}
\ No newline at end of file
+  "CHAIN_ID": "1",
+  "ENVIRONMENT": "production",
+  "AUTH_URL": "/auth",
```

Comment:
this shouldn't be `/auth/login` ? 
Not seeing this route on the server

Reply:
the `auth dApp` works under the `/auth` path, the `login` endpoint is concatenated when required in the login flow